src/k2/httpproxy/HTTPProxy.h

```diff
@@ -109,6 +119,9 @@ class HTTPProxy {
         std::unordered_map<String,
             std::unordered_map<uint32_t, std::shared_ptr<shd::Schema>>
     >> _shdSchemas;
+    ExpiryList<ManagedTxn, &ManagedTxn::tsLink> _expiryList;
+    // Txn idle timeout
+    ConfigDuration httpproxy_txn_timeout{"httpproxy_txn_timeout", 60s};
```

Comment:
please use camelCase for variables, not underscores. Also, this is a private variable so it should start with "_"